docs/content/guides/dagster/ml-ops.mdx

```diff
@@ -0,0 +1,221 @@
+
+In our example, we will be training a [XGBoost model](https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.XGBRegressor) to predict a numerical value.
+
+```python file=/guides/dagster/ml_ops/ml_ops.py startafter=models_start endbefore=models_end
+import xgboost as xg
+from sklearn.metrics import mean_absolute_error
+
+@asset
+def xgboost(transformed_training_data):
+    transformed_X_train, transformed_y_train = transformed_training_data
+    """Train XGBoost model, which is a highly efficent and flexible model"""
+    xgb_r = xg.XGBRegressor(objective ='reg:squarederror', eval_metric=mean_absolute_error,
+                  n_estimators = 20)
+    xgb_r.fit(transformed_X_train, transformed_y_train)
+    return xgb_r
+
+@asset
+def score_xgboost( transformed_test_data, xgboost):
+    transformed_X_test, transformed_y_test = transformed_test_data
+    """Use the test set data to get a score of the XGBoost model"""
+    score = xgboost.score(transformed_X_test, transformed_y_test)
+    return score
+```
+
+We include freshness policies in the models here which will ensure the upstream data transformation assets are updated to comply with these policies.
+
+### Evaluate our results
+
+In our model assets, we evaluated each of the models on the test data and in this case, got the error between the predicted and actual results. Next, lets create an asset which runs inference on the model, more frequently than the model is re-trained to predict the results.
+
+```python file=/guides/dagster/ml_ops/ml_ops.py startafter=inference_start endbefore=inference_end
+@asset(freshness_policy=FreshnessPolicy(maximum_lag_minutes=60))
+def model_inference(xgboost, Tfidf_Vectorizer):
+    """Get the max ID number from hacker news"""
+
+    latest_item = requests.get(
+        f"https://hacker-news.firebaseio.com/v0/maxitem.json"
+    ).json()
+    """Get items based on story ids from the HackerNews items endpoint"""
+    results = []
+    scope = range(latest_item-100, latest_item)
+    for item_id in scope:
+        item = requests.get(
+            f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+        ).json()
+        results.append(item)
+
+    df = pd.DataFrame(results)
+    if len(df) > 0:
+        df = df[df.type == "story"]
+        df = df[~df.title.isna()]
+    inference_x = df.title
+    inference_x = Tfidf_Vectorizer.transform(inference_x)
+    return xgboost.predict(inference_x)
+```
+
+Depending on what the objective of your ML model is, you can use this data to set alerts, save model performance history and trigger retraining.
+
+Dagster integrates with [Weights & Biases](/\_apidocs/libraries/dagster-wandb) and an [example](https://github.com/dagster-io/dagster/tree/master/examples/with_wandb) which demonstrates how to use W\&B's artifacts with Dagster.
+
+---
+
+## Machine learning operations
```

Comment:
My instinct would be to leave this out for the initial PR.  I think there's value in this kind of content, and I think this is a solid start, but I also think it will take some back and forth to get it right, and I'd rather not have this block the rest of the guide from making it in.

Reply:
sounds good saving it in notion for later